app/src/main/java/com/stevesoltys/seedvault/transport/ConfigurableBackupTransportService.kt

```diff
@@ -63,7 +63,7 @@ fun requestBackup(context: Context) {
     val observer = NotificationBackupObserver(context, packages.size, appTotals)
     val result = try {
         val backupManager: IBackupManager = get().koin.get()
-        backupManager.requestBackup(packages, observer, BackupMonitor(), FLAG_USER_INITIATED)
+        backupManager.requestBackup(packages, observer, BackupMonitor(), 0)
```

Comment:
@grote was this the issue with the flag and incremental backup you were talking about?

Reply:
Yes, exactly. See also the commit message of this commit.